--- rrdtool.c	2005-12-03 14:34:06.000000000 +0100
+++ rrdtool.c.oden	2012-05-06 11:40:51.720522819 +0200
@@ -39,7 +39,7 @@
 ZEND_DECLARE_MODULE_GLOBALS(rrdtool)
  */
  
-function_entry rrdtool_functions[] = {
+zend_function_entry rrdtool_functions[] = {
 	PHP_FE(rrd_graph,				NULL)
 	PHP_FE(rrd_fetch,				NULL)
 	PHP_FE(rrd_error,				NULL)
@@ -122,8 +122,8 @@ PHP_MINFO_FUNCTION(rrdtool)
 	Creates a graph based on options passed via an array */
 PHP_FUNCTION(rrd_graph)
 {
-	pval *file, *args, *p_argc;
-	pval *entry;
+	zval *file, *args, *p_argc;
+	zval *entry;
 	zval *p_calcpr;
 	HashTable *args_arr;
 	int i, xsize, ysize, argc;
@@ -157,7 +157,7 @@ PHP_FUNCTION(rrd_graph)
 
 		for (i = 3; i < argc; i++) 
 		{
-			pval **dataptr;
+			zval **dataptr;
 
 			if ( zend_hash_get_current_data(args_arr, (void *) &dataptr) == FAILURE )
 				continue;
@@ -221,9 +221,9 @@ PHP_FUNCTION(rrd_graph)
 	Fetch info from an RRD file */
 PHP_FUNCTION(rrd_fetch)
 {
-	pval *file, *args, *p_argc;
-	pval *entry;
-	pval *p_start, *p_end, *p_step, *p_ds_cnt;
+	zval *file, *args, *p_argc;
+	zval *entry;
+	zval *p_start, *p_end, *p_step, *p_ds_cnt;
 	HashTable *args_arr;
 	zval *p_ds_namv, *p_data;
 	int i, j, argc;
@@ -259,7 +259,7 @@ PHP_FUNCTION(rrd_fetch)
 
 		for (i = 3; i < argc; i++) 
 		{
-			pval **dataptr;
+			zval **dataptr;
 
 			if ( zend_hash_get_current_data(args_arr, (void *) &dataptr) == FAILURE )
 				continue;
@@ -366,7 +366,7 @@ PHP_FUNCTION(rrd_clear_error)
 	Update an RRD file with values specified */
 PHP_FUNCTION(rrd_update)
 {
-	pval *file, *opt;
+	zval *file, *opt;
 	char **argv;
 
 	if ( rrd_test_error() )
@@ -409,7 +409,7 @@ PHP_FUNCTION(rrd_update)
 	Gets last update time of an RRD file */
 PHP_FUNCTION(rrd_last)
 {
-	pval *file;
+	zval *file;
 	unsigned long retval;
 
 	char **argv = (char **) emalloc(3 * sizeof(char *));
@@ -444,8 +444,8 @@ PHP_FUNCTION(rrd_last)
 	Create an RRD file with the options passed (passed via array) */ 
 PHP_FUNCTION(rrd_create)
 {
-	pval *file, *args, *p_argc;
-	pval *entry;
+	zval *file, *args, *p_argc;
+	zval *entry;
 	char **argv;
 	HashTable *args_arr;
 	int argc, i;
@@ -478,7 +478,7 @@ PHP_FUNCTION(rrd_create)
 
 		for (i = 3; i < argc; i++) 
 		{
-			pval **dataptr;
+			zval **dataptr;
 
 			if ( zend_hash_get_current_data(args_arr, (void *) &dataptr) == FAILURE )
 				continue;
